subgraph.template.yaml

```diff
@@ -152,3 +152,25 @@ dataSources:
           handler: handleCreateSRRWithProof
         - event: MigrateBatch(indexed bytes32,bytes32[],address,uint256,uint256)
           handler: handleMigrateBatch
+  - kind: ethereum/contract
```

Comment:
So, we will have a new contract entry in that case I think deployment.json should have contract entry as well.

Reply:
@noyonthe1 thanks, I added the BulkTransfer into deployments.json
https://github.com/startbahn/startrail-subgraph/pull/64/files#diff-0459c08daaef16ee190d4991e600a598e34e76a05bd2d122c38030ad040ebea1R15

Reply:
oh I see, sorry I missed that part then.